### PR TITLE
MRG, DOC: Add data channels to glossary

### PR DESCRIPTION
**Pull request by larsoner**
Adds `data channels` to our glossary with a couple of refs (can add more later as we find them). Also clarifies a bit what happens by default with `create_info`, as these two conspired to trip up @arokem on Gitter.

@drammock can you look at the output and see if you'd like to tweak, then merge?

### CODE REVIEW

**Comment by larsoner** [comment]
This is what the output looks like locally:

Screenshot from 2020-03-26 17-08-38

**Comment by larsoner** [comment]
Setting to WIP because I realized that we should also provide the mapping to human-readable names in there...

**Comment by drammock** [comment]
Based on your screenshot, the definition and the list don't jive. It looks like you just listed all channel types.

**Comment by larsoner** [comment]
Indeed misc is in there. These private variables need some renaming...

**Comment by larsoner** [comment]
[Should be fixed](https://18964-1301584-gh.circle-artifacts.com/0/dev/glossary.html#term-data-channels)

Also I deprecated `ch_types=None` because there is no reason to have it instead of just `ch_types='misc'`.

File: mne/defaults.py [hunk 1]
```diff
@@ -37,8 +37,8 @@
     titles=dict(mag='Magnetometers', grad='Gradiometers', eeg='EEG', eog='EOG',
                 ecg='ECG', emg='EMG', misc='misc', seeg='sEEG', bio='BIO',
                 dipole='Dipole', ecog='ECoG', hbo='Oxyhemoglobin',
-                ref_meg='Reference Magnetometers', fnirs_raw='fNIRS',
-                fnirs_od='fNIRS', hbr='Deoxyhemoglobin',
+                ref_meg='Reference Magnetometers', fnirs_raw='fNIRS (raw)',
+                fnirs_od='fNIRS (OD)', hbr='Deoxyhemoglobin',
```
Comment (larsoner) [inline mne/defaults.py]: @rob-luke okay for you?

Comment (rob-luke) [reply mne/defaults.py]: All good by me thanks :)

**Pull request merged**